### ISSUE TITLE
Bug Report - Increased training time

### ISSUE THREAD

**Issue by RandomKaos999** [kohya-ss/musubi-tuner#238]
Hi there,

Recently started using this to tinker with training loras with Hunyuan, but anyway, I seem to of come across an somewhat interesting bug or issue.

On a Windows 11 system, if I have the --output_dir variable set with a slash on the end, ie "X:\test\" then the training time is increased MASSIVELY.

Copy and paste example: 25/750 [1:22:03<39:39:50, 196.95s/it, avr_loss=0.194]

This is running on a 3060 with blocks to swap at 18 btw. If I remove the trailing slash from the file path, so its just "X:\test":

30/750 [04:53<1:57:24,  9.78s/it, avr_loss=0.172]

Just thought you would want to know.

For the record, im an idiot for not noticing, but also I've just realised ive wasted 5 hours of my life testing different scenarios and datasets for something so trivial :(

**Comment by RandomKaos999** [comment]
That was the exact reason why I didnt even consider it as a problem during the actual training phase and it drove me crazy.
This was the command line used for the training. The only thing changed was the output_dir option and the final slash removed. Oh ignore the training rate, tinkering with it at the moment, but originally was 2e-4.

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 hv_train_network.py --dit "F:\GenAI\Models\HunyuanTrainin\transformers\mp_rank_00_model_states_fp8.safetensors" --dataset_config "<edited>" --sdpa --mixed_precision bf16 --fp8_base --optimizer_type adamw8bit --learning_rate 5e-4 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora --network_dim 32 --timestep_sampling shift --discrete_flow_shift 7.0 --max_train_epochs 30 --save_every_n_epochs 1 --seed 1234 --output_dir "F:\.....\model\" --output_name rs1-videoloratest-images --blocks_to_swap 24

**Comment by RandomKaos999** [comment]
Currently training run with the above command line:

steps:  30%|█████████████████████████▌                                                           | 225/750 [36:17<1:24:39,  9.68s/it, avr_loss=0.179]

Cancelled it, quickly added the \ on the end of output_dir again with the above- waited for the first step to complete - For the record:

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 hv_train_network.py --dit "F:\GenAI\Models\HunyuanTrainin\transformers\mp_rank_00_model_states_fp8.safetensors" --dataset_config "" --sdpa --mixed_precision bf16 --fp8_base --optimizer_type adamw8bit --learning_rate 5e-4 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora --network_dim 32 --timestep_sampling shift --discrete_flow_shift 7.0 --max_train_epochs 30 --save_every_n_epochs 1 --seed 1234 --output_dir "F:.....\model\" --output_name rs1-videoloratest-images --blocks_to_swap 24

(Just the trailing slash on --output_dir added):

First step:

steps:   0%|                                                                                      | 1/750 [03:05<38:31:58, 185.21s/it, avr_loss=0.26]

**Comment by RandomKaos999** [comment]
Cancelled again, removed the trailing slash:

steps:   0%|                                                                                        | 1/750 [00:16<3:25:39, 16.48s/it, avr_loss=0.26]
